### PR TITLE
[API documentation] How to load Loris with candidate/instrument data

### PR DESCRIPTION
**Pull request by mohini-tripathi**
## Brief summary of changes

- [x] Have you updated related documentation?

TASKS DONE:
- Added link in LORIS repo Readme to Xavier's existing Jupyter tutorial on how to use the API to download/upload data.
- Created FAQ File.

Any modifications/ typos in the FAQ will be updated as per suggestions. 

#### Link(s) to related issue(s)

* Resolves # This fixes https://github.com/aces/Loris/issues/4471
#GSOC Aspirant
CC @christinerogers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,13 @@ If you use MySQL 8, please read [this link](https://www.php.net/manual/en/mysqli
 
 7. Follow the [Setup Guide in the LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) to complete your post-installation setup and configuration, and for more documentation.
 
+## Loris API
+Loris API is intended to provide a stable, versioned REST interface for interacting with the core LORIS functionality without requiring backend database access.
+
+### Usage
+Follow our Jupyter Notebook guide on [How to use Loris API to download/upload data](https://github.com/xlecours/loris-api-presentation). For any query, please read [FAQs](https://github.com/aces/Loris/blob/API-doc-4471/modules/api/docs/FAQ.md).
```
[the remark this answers is not in the document]
Comment (mohini-tripathi) [reply README.md:103]: I will remove the FAQs link from the doc. Information regarding configuring Loris will be added to the FAQ, for now, I am removing it. Thanks for your suggestion.

File: README.md [hunk 2]
```diff
@@ -96,6 +96,13 @@ If you use MySQL 8, please read [this link](https://www.php.net/manual/en/mysqli
 
 7. Follow the [Setup Guide in the LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) to complete your post-installation setup and configuration, and for more documentation.
 
+## Loris API
+Loris API is intended to provide a stable, versioned REST interface for interacting with the core LORIS functionality without requiring backend database access.
+
+### Usage
+Follow our Jupyter Notebook guide on [How to use Loris API to download/upload data](https://github.com/xlecours/loris-api-presentation).
```
Comment (johnsaigle) [inline README.md:103]:
```suggestion
Follow our Jupyter Notebook guide on [How to use LORIS API to download/upload data](https://github.com/xlecours/loris-api-presentation).
```

File: README.md [hunk 3]
```diff
@@ -96,6 +96,13 @@ If you use MySQL 8, please read [this link](https://www.php.net/manual/en/mysqli
 
 7. Follow the [Setup Guide in the LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) to complete your post-installation setup and configuration, and for more documentation.
 
+## Loris API
```
Comment (johnsaigle) [inline README.md:99]:
```suggestion
## LORIS API
```

File: README.md [hunk 4]
```diff
@@ -96,6 +96,13 @@ If you use MySQL 8, please read [this link](https://www.php.net/manual/en/mysqli
...
+Loris API is intended to provide a stable, versioned REST interface for interacting with the core LORIS functionality without requiring backend database access.
```
Comment (johnsaigle) [inline README.md:100]:
```suggestion
LORIS API is intended to provide a stable, versioned REST interface for interacting with the core LORIS functionality without requiring backend database access.
```

**Comment by driusan** [comment]
This is linking to a third party repo which isn't part of "our" documentation as the text claims. @xlecours is looking into moving the Jupyter Notebook into our actual documentation rather than his repo, and will link it from the appropriate places when he does.

**Pull request closed without merging**